Sources/Rendering/Core/Renderer/index.js

```diff
@@ -181,9 +181,13 @@ function vtkRenderer(publicAPI, model) {
     publicAPI.modified();
   };
 
+  publicAPI.hasLight = (light) =>
+    !!model.lights.filter((item) => item === light).length;
```

Comment:
This can be written as `model.lights.includes(light)`.

Reply:
Cool. I was copying this from the [the view prop one](https://github.com/Kitware/vtk-js/blob/24d26e040e3454f61b882ab7d95982d084f03a9d/Sources/Rendering/Core/Viewport/index.js#L19-L25). Should I change that one too?

Reply:
Yeah sure, that'll be good to simplify as well.

Reply:
Done.